src/lib/childrenUtils.js

```diff
@@ -16,3 +16,14 @@ export const someByType = (children, type) => _.some(Children.toArray(children),
  * @returns {undefined|Object}
  */
 export const findByType = (children, type) => _.find(Children.toArray(children), { type })
+
+/**
+ * Tests if any children.
+ * @param {Object} children The children prop of a component.
+ * @returns {Boolean}
+ */
+export const isNil = (children) => {
```

Comment:
Since this is now a very critical util for the lib, let's get some tests on it.  You can see other `lib` tests in `test/specs/lib`.